subsystems/win/basesrv/dosdev.c

```diff
@@ -146,6 +158,317 @@ IsGlobalSymbolicLink(HANDLE LinkHandle,
     return Status;
 }
 
+BOOLEAN
+CheckForGlobalDriveLetter(SHORT DriveLetter)
+{
+    WCHAR Path[8];
+    NTSTATUS Status;
+    BOOLEAN IsGlobal;
+    UNICODE_STRING PathU;
+    HANDLE SymbolicLinkHandle;
+    OBJECT_ATTRIBUTES ObjectAttributes;
+
+    /* Setup our drive path */
+    wcsncpy(Path, L"\\??\\X:", (sizeof(L"\\??\\X:") / sizeof(WCHAR)));
```

Comment:
This seems like an unnecessarily complicated way of doing `wcscpy`.
I would of course prefer RtlStringCbCopyW :p